src/routes/Player/ControlBar/SeekBar/formatTime.js

```diff
@@ -4,15 +4,15 @@ const formatUnit = (value) => {
     return ('0' + value).slice(-1 * Math.max(value.toString().length, 2));
 };
 
-const formatTime = (time) => {
+const formatTime = (time, prefix = '') => {
     if (time === null || isNaN(time)) {
         return '--:--:--';
     }
 
     const hours = Math.floor(time / (1000 * 60 * 60));
     const minutes = Math.floor((time / (1000 * 60)) % 60);
     const seconds = Math.floor((time / 1000) % 60);
-    return `${formatUnit(hours)}:${formatUnit(minutes)}:${formatUnit(seconds)}`;
+    return `${formatUnit(hours)}:${formatUnit(minutes)}:${formatUnit(seconds)}${prefix}`;
```

Comment:
This `prefix` is at that end because `direction` for this component is `rtl` in `src/routes/Player/ControlBar/SeekBar/styles.less`